src/components/Uploader/index.js

```diff
@@ -4,22 +4,29 @@ import theme from '../Theme/style.css'
 import style from './style.css'
 import {errors} from '../strings/errors'
 import { trackComponentAndMode } from '../../Tracker'
+import SwitchDevice from '../crossDevice/SwitchDevice'
+import { isDesktop } from '../utils'
+import { mobileCopy, desktopCopy } from '../strings/uploadCopy'
 
-const UploadInstructions = ({error}) =>
+const instructionsCopy = (method, side) => {
+  const instructions = isDesktop ? desktopCopy.instructions : mobileCopy.instructions
+   return method === 'document' ? instructions[method][side] : instructions[method]
+}
+
+const UploadInstructions = ({error, method, side}) =>
   <div className={style.base}>
     <span className={`${theme.icon} ${style.icon}`}></span>
-    <p className={style.text}>Take a photo with your camera or upload one from your library.</p>
+    <p className={style.text}>{instructionsCopy(method, side)}</p>
     <UploadError error={errors[error.name]} />
   </div>
 
+
 const UploadError = ({error}) =>
   error && <div className={`${style.text} ${style.error}`}>{`${error.message}. ${error.instruction}.`}</div>
 
-const MobileLink = ({mobileUrl}) =>
-  mobileUrl && <p className={style.mobileUrl}>Mobile: {mobileUrl}</p>
-
-const UploaderPure = ({onImageSelected, error, mobileUrl}) =>
+const UploaderPure = ({method, side, onImageSelected, error, ...{changeFlow}}) =>
   <div>
+    { isDesktop ? <SwitchDevice {...{changeFlow}} /> : '' }
```

Comment:
Not sure if `isDesktop` is the most correct way of conditionally showing/not showing this.
Since it's not very reliable and reality is, we never want to show this if you are on cross device link.